### PR TITLE
366 Add doc of how to use 'NOT'

### PR DESCRIPTION
**Pull request by BibianaC**
I have updated `help.html`. There is also `advance_search.html` I don't see it being used anywhere.

### CODE REVIEW

File: grantnav/frontend/templates/help.html [hunk 1]
```diff
@@ -240,6 +240,7 @@ <h3 id="words_phrases">{% trans "Words and phrases" %}</h3>
     <li>{% trans "Multiple words: e.g. <code>young people gardens</code> (each word does not necessarily have to be present)" %}</li>
     <li>{% trans "Require each word to be found: e.g. <code>young AND people</code>" %}</li>
     <li>{% trans 'An exact phrase by enclosing it in quotes: e.g. <code>"young people"</code>' %}</li>
+    <li>{% trans 'Exclude an organisation: e.g. <code>NOT fundingOrganization.name:"London Councils"</code>' %}</li>
```
Comment (robredpath) [inline grantnav/frontend/templates/help.html]: Maybe "Exclude results that contain a word: e.g. youth NOT clubs"

Comment (BibianaC) [reply grantnav/frontend/templates/help.html]: @robredpath I have updated it.
What about `advanced_search.html`, is it being used?

**Comment by robredpath** [comment]
@BibianaC yeah, it looks like `advanced_search.html` has been rolled into the main help page. You can remove it.

**Comment by BibianaC** [comment]
@robredpath I have deleted the file.

**Comment by robredpath** [comment]
Great! @morchickit - can you take a look at this as a solution to #366 ?

**Comment by morchickit** [comment]
Let's commit it. :-)

**Comment by morchickit** [comment]
Sorry, I meant let's push it!

**Pull request merged**